evalml/automl/automl_search.py

```diff
@@ -1331,7 +1347,7 @@ def load(file_path):
             AutoSearchBase object
         """
         with open(file_path, "rb") as f:
-            return cloudpickle.load(f)
+            return pickle.load(f)
```

Comment:
So we can save with cloudpickle and read with pickle?! Wow, I had no idea that works hehe.

I feel like we should accept an argument here for the "pickle type"? Feels weird to offer a choice of library for `save` but not respect that in `load`. Not blocking though.

Reply:
Hmm yeah I agree, it does feel symmetrical to do so. I think it would be a no-op though since it looks like the doc for cloudpickle just recommends using the standard python pickler for loading.